bitwarden_license/src/Sso/Controllers/AccountController.cs

```diff
@@ -148,37 +146,65 @@ IActionResult invalidJson(string errorMessageKey, Exception ex = null)
                     {
                         errorKey = ex.Message;
                     }
-                    return invalidJson(errorKey, translatedException.ResourceNotFound ? ex : null);
+                    return InvalidJson(errorKey, translatedException.ResourceNotFound ? ex : null);
                 }
+
+                var ssoToken = new SsoToken()
+                {
+                    DomainHint = domainHint,
+                    OrganizationId = organization.Id
```

Comment:
Code Style: Should have a trailing comma.
```suggestion
                    OrganizationId = organization.Id,
```